apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -140,6 +163,49 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
         setSelectedUserStore(userstore);
         getUserStoreList();
     }, [ userstore ]);
+    const [ configureMode, setConfigureMode ] = useState<string>(undefined);
+    const [ emailData, setEmailData ] = useState<string[]>();
+    const [ isEmailDataError, setIsEmailDataError ] = useState<boolean>(false);
+    const [ emailDataError, setEmailDataError ] = useState<string>("");
+    const [ userStoreRegex, setUserStoreRegex ] = useState<string>("");
+    const [ regExLoading, setRegExLoading ] = useState<boolean>(false);
+
+    const { data: validationData } = useValidationConfigData();
+
+    const config: ValidationFormInterface = getUsernameConfiguration(validationData);
+    const isAlphanumericUsername: boolean = config?.enableValidator === "true" ? true : false;
+    const optionsArray: string[] = [];
+
+    /**
+     * Get the userstore.
+     */
+    useEffect(() => {
+        getUserStoreRegex();
```

Comment:
Can't we define this inline? Since the function is not used anywhere else

Reply:
Fixed with 899a8e06a8a09eccfe9a0f5feb3405a95bcae33e.